README.md

```diff
@@ -1,20 +1,30 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
 
-(b) Where in your JS file should you put it, and WHY? 
+A click handler is an event listener that will fire off a function once the target has been clicked.
+
+(b) Where in your JS file should you put it, and WHY?
+
+You should put in the document ready. You need to make sure the target element exists before you attach anything to it.
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+
+It means that $ is undefined. idk, probably means JQuery isn't being loaded in.
 
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
+What should go in the `$(document).ready()` part of your JS file?
+
+Anything that needs to reference a specific DOM element.
```

Comment:
Yes, and more specifically, anything that needs to reference a specific DOM element *that already exists when the page initially loads.* If you create new buttons after the page loads, for example, the click handlers for those should not go in `$(document).ready()`.